site/src/pages/HealthPage/DERPPage.tsx

```diff
@@ -85,7 +85,11 @@ export const DERPPage: FC = () => {
 				<section>
 					<SectionLabel>Regions</SectionLabel>
 					<div css={{ display: "flex", flexWrap: "wrap", gap: 12 }}>
-						{Object.values(regions)
+						{Object.values(regions!)
```

Comment:
Not a huge problem, but this `!` isn't doing anything. At least from the type level, `regions` itself can't be nullish